commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -147,9 +147,11 @@
     <li class=change-in-release> Add field modified a to Payload {@link io.sphere.sdk.subscriptions.ResourceCreatedPayload}, {@link io.sphere.sdk.subscriptions.ResourceUpdatedPayload} and {@link io.sphere.sdk.subscriptions.ResourceDeletedPayload} </li>
     <li class=new-in-release> Now its possible to query the subscriptions endpoint health via the new {@link Subscription#getStatus()}</li>
     <li class=change-in-release>Added new attributes  {@link OrderFromCartDraft#getOrderState()} and {@link OrderFromCartDraft#getState()}</li>
-    <li class=new-in-release>new update action {@link io.sphere.sdk.productdiscounts.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
+    <li class=new-in-release>New update action {@link io.sphere.sdk.productdiscounts.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
     <li class=change-in-release>{@link io.sphere.sdk.producttypes.commands.updateactions.AddAttributeDefinition#of(AttributeDefinition)} changed parameter to {@link io.sphere.sdk.producttypes.commands.updateactions.AddAttributeDefinition#of(AttributeDefinitionDraft)} instead</li>
-    <li class=new-in-release>now possible to change customer in order via {@link io.sphere.sdk.orders.commands.updateactions.SetCustomerId} which would result in a {@link io.sphere.sdk.orders.messages.OrderCustomerSetMessage}</li>
+    <li class=new-in-release>It's now possible to change customer in order via {@link io.sphere.sdk.orders.commands.updateactions.SetCustomerId} which would result in a {@link io.sphere.sdk.orders.messages.OrderCustomerSetMessage}</li>
+    <li  class=new-in-release>New update action {@link io.sphere.sdk.cartdiscounts.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
+    <li  class=new-in-release>New update action {@link io.sphere.sdk.discountcodes.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
```

Comment:
Both notes say the same, was it maybe CartDiscount?

Reply:
Good catch 👍 Fixed: should be `CartDiscount` and `DiscountCode`.

Reply:
Good catch 👍 Fixed: should be `CartDiscount` and `DiscountCode`.